### ISSUE TITLE
OG-ZAF not compatible with OG-Core 0.11.13

### ISSUE THREAD

**Issue by rickecon** [EAPD-DRB/OG-ZAF#59]
@jdebacker. I just recreated the conda environment `ogzaf-dev` that included the new version of OG-Core (v0.11.13). I ran the `run_og_zaf.py` file and got the following error. I think this is coming from the new pension adjustments.
```
(ogzaf-dev) richardevans@Richards-MacBook-Pro-2 examples % python run_og_zaf.py
Number of workers =  7
Traceback (most recent call last):
  File "/Users/richardevans/Docs/Economics/OSE/OG-ZAF/examples/run_og_zaf.py", line 139, in <module>
    main()
  File "/Users/richardevans/Docs/Economics/OSE/OG-ZAF/examples/run_og_zaf.py", line 49, in main
    p.update_specifications(
  File "/opt/anaconda3/envs/ogzaf-dev/lib/python3.11/site-packages/ogcore/parameters.py", line 440, in update_specifications
    self.adjust(revision, raise_errors=raise_errors)
  File "/opt/anaconda3/envs/ogzaf-dev/lib/python3.11/site-packages/paramtools/parameters.py", line 257, in adjust
    return self._adjust(
           ^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/ogzaf-dev/lib/python3.11/site-packages/paramtools/parameters.py", line 375, in _adjust
    raise self.validation_error
paramtools.exceptions.ValidationError: {
    "errors": {
        "schema": [
            "Unknown field: AIME_num_years"
        ]
    }
}

```

**Comment by jdebacker** [comment]
@rickecon Yes, this is due to a parameter name change: `AIME_num_years` -> `avg_earn_num_years`.

**Comment by jdebacker** [comment]
Addressed via PR #60
